### PR TITLE
Create participant data model

### PR DESCRIPTION
**Pull request by rqding**
1. Remove the old project data class model
2. Add participant data model under the project app
3. Add django admin function to add, remove, update, and list the participant data
4. Add django admin function to control only show the participant data the user is a project team member
5. Add puml file as the database diagram

### CODE REVIEW

File: django/project_app/admin.py [hunk 1]
```diff
@@ -4,14 +4,25 @@
...
+from base_app.models import Project
+from .models import Participant
 
 
-@admin.register(Project_data)
-class ProjectDataAdmin(admin.ModelAdmin):
-    list_display = ('id', 'project_name', 'label', 'description')
-    search_fields = ('project__name', 'label', 'description')
-    ordering = ('id',)
+class ParticipantAdmin(admin.ModelAdmin):
```
Comment (rqding) [inline django/project_app/admin.py:11]: This admin function will list all the information of the participant table

File: django/project_app/admin.py [hunk 2]
```diff
@@ -4,14 +4,25 @@
...
+    list_display = (
+    'participant_id', 'project', 'gender', 'date_of_birth', 'race',
+    'marital_status', 'language')
+    list_filter = ('project',)
 
-    def project_name(self, obj):
-        return obj.project.name
\ No newline at end of file
+    def get_queryset(self, request):
```
Comment (rqding) [inline django/project_app/admin.py:17]: This function is to limit the data user could view from django admin page. User should only able to view the data if they are the team member of a project

**Comment by rqding** [comment]
For this Pull request, you could focus on the design.md file which is puml file of ER diagram of the relationship between the project table and the participant table. The model.py file is the key data model file. The admin.py file provides the function of how django admin will work.

**Comment by lwjohnst86** [comment]
I tried to update the migrations by running `python3 manage.py makemigrations` but it threw an error about  not importing urlpatterns from project_app.urls... @rqding, how do we fix that?

**Comment by rqding** [comment]
I think it should be python manage.py makemigratios, because the python command already points to the python 3 versions. But if it is not helping, please send me the error message

**Pull request merged**